src/olympia/abuse/tests/test_models.py

```diff
@@ -2356,7 +2337,7 @@ def test_get_action_helper(self):
                     'rating': None,
                     'collection': None,
                     'user': None,
-                    **targets[ActionClass],
+                    **targets.get(ActionClass, {'addon': addon}),
```

Comment:
Why? This is a big behavior change as the index accessor will raise if the key is not defined, now instead we default to addon? Could you add an explainer why we default to addon: addon?

Reply:
Add as code comments?  It was mainly as I kept having to add new `ContentAction...` classes to the dict but they all ended up using `addon` so it was redundant.  Made sense to me, but do you think it should remain an explicit list of all the possible targets?